conda/recipes/librmm/meta.yaml

```diff
@@ -97,7 +98,7 @@ outputs:
     script: install_librmm_tests.sh
     build:
       number: {{ GIT_DESCRIBE_NUMBER }}
-      string: cuda{{ cuda_major }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+      string: cuda{{ cuda_major }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}{{ date_string }}
```

Comment:
I would add an underscore between `GIT_DESCRIBE_NUMBER` and `date_string` so you can easily tell them apart.

Reply:
Whoops I see this is in the definition above.